core/sqf/src/seatrans/tm/hbasetmlib2/src/main/java/org/trafodion/dtm/TmAuditTlog.java

```diff
@@ -961,18 +814,14 @@ public static boolean deleteRecord(final long lvTransid) throws IOException {
       if (LOG.isTraceEnabled()) LOG.trace("deleteRecord start " + lvTransid);
       String transidString = new String(String.valueOf(lvTransid));
       int lv_lockIndex = (int)(lvTransid & tLogHashKey);
-      try {
+      //try {
```

Comment:
Maybe should remove this comment? (since the corresponding catch block was removed)